### PR TITLE
Changing add group wizard header text

### PR DESCRIPTION
**Pull request by tahmidefaz**
Jira: https://projects.engineering.redhat.com/browse/RHCLOUD-4441
**Screenshot**
image

### CODE REVIEW

**Comment by tahmidefaz** [comment]
Ping! @mmenestr

File: src/smart-components/group/add-group/add-group-wizard.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ const AddGroupWizard = ({
   return (
     <Wizard
       isLarge
-      title={ 'Add group' }
+      title={ 'Create and configure a group' }
```
Comment (karelhala) [inline src/smart-components/group/add-group/add-group-wizard.js]: No need to wrap in object, you can directly write `title="Create and configure a group"` same for description

Comment (tahmidefaz) [reply src/smart-components/group/add-group/add-group-wizard.js]: Oh, thanks! Totally missed it, just followed the code pattern

**Comment by mmenestr** [comment]
@tahmidefaz Perfect!

**Pull request merged**